server/pruning.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"github.com/cosmos/cosmos-sdk/store"
+	"github.com/spf13/viper"
+)
+
+//GetPruningOptionsFromFlags parses start command flags and returns the correct PruningOptions.
```

Comment:
```suggestion
// GetPruningOptionsFromFlags parses start command flags and returns the correct PruningOptions.
```

Reply:
also mention that: 
- `flagPruning` prevails over `flagPruningKeepEvery` and `flagPruningSnapshotEvery`.
- Default option is `PruneSyncable`

Reply:
++